packages/mermaid/src/docs/community/security.md

```diff
@@ -4,13 +4,13 @@ The Mermaid team takes the security of Mermaid and the applications that use Mer
 
 ## Reporting vulnerabilities
 
-To report a vulnerability, please e-mail security@mermaid.live with a description of the issue, the steps you took to create the issue, affected versions, and if known, mitigations for the issue.
+To report a vulnerability, please e-mail <security@mermaid.live> with a description of the issue, the steps you took to create the issue, affected versions, and if known, mitigations for the issue.
 
 We aim to reply within three working days, probably much sooner.
 
-You should expect a close collaboration as we work to resolve the issue you have reported. Please reach out to security@mermaid.live again if you do not receive prompt attention and regular updates.
+You should expect a close collaboration as we work to resolve the issue you have reported. Please reach out to <security@mermaid.live> again if you do not receive prompt attention and regular updates.
 
-You may also reach out to the team via our public Slack chat channels; however, please make sure to e-mail security@mernaid.live when reporting an issue, and avoid revealing information about vulnerabilities in public as that could that could put users at risk.
+You may also reach out to the team via our public Slack chat channels; however, please make sure to e-mail <security@mermaid.live> when reporting an issue, and avoid revealing information about vulnerabilities in public as that could that could put users at risk.
```

Comment:
There was a type in the email address here.

Also I changed the formatting of the email addresses. 

I did not check yet if this actually works with Vitepress.
Should probably do that :)

Reply:
Good catch on that typo.

If it doesn't work in Vitepress, we could also make it a `mailto:` uri, see https://en.wikipedia.org/wiki/Mailto, something like:

```suggestion
You may also reach out to the team via our public Slack chat channels; however, please make sure to e-mail [security@mermaid.live](mailto:security@mermaid.live) when reporting an issue, and avoid revealing information about vulnerabilities in public as that could that could put users at risk.
```

Reply:
I have confirmed that the emails get rendered correctly and also get the `mailto:` prefix when running the docs locally. See screenshot. So we can leave this as is.

![Screenshot 2022-11-07 at 10 14 35 ](https://user-images.githubusercontent.com/163029/200273169-eabb449d-a74a-430e-a2f8-a323aeb50309.png)